src/Curl.Converter.Net/README.md

```diff
@@ -0,0 +1,53 @@
+# Curl.Converter.Net
```

Comment:
I'd suggest keeping a single readme file at the root of the repo for all projects.

Reply:
No problem. I merged everything in the main README.md